CHANGES.md

```diff
@@ -24,7 +24,12 @@ OpenSSL 3.1
 
 ### Changes between 3.1.1 and 3.1.2 [xx XXX xxxx]
 
- * none yet
+ * When building with the `enable-fips` option and using the resulting
+   FIPS provider, TLS 1.2 will mandate the use of an extended master
+   secret and the Hash and HMAC DRBGs will not operate with truncated
+   digests.
```

Comment:
This text contradicts the one from the NEWS - the NEWS one is mentioning default settings, and this one says about enforcing.

Reply:
_building with the `enable-fips` option and using the resulting FIPS provider_ is equivalent to my mind.

Reply:
The NEWS wording didn't seem appropriate to the CHANGES file.
NEWS is more verbose and explanatory IMO.

Reply:
> The NEWS wording didn't seem appropriate to the CHANGES file. NEWS is more verbose and explanatory IMO.

Nope? CHANGES.md entries are always the more verbose ones.

Reply:
But IMO this text is fine and could go into both CHANGES.md and NEWS.md.